fastlane/Fastfile

```diff
@@ -89,7 +89,26 @@ platform :ios do
       ],
       submit_beta_review: true,
       notify_external_testers: true,
-      expire_previous_builds: true,
+      changelog: commit[:message]
+    )
+  end
+
+  desc "Publish a beta release to the user study TestFlight group"
+  lane :userstudy do
```

Comment:
@jdisho  I added a separate fastlane group and disabled the invalidation of old builds for the different user groups to ensure that we can have the two release trains in parallel.